diff --git a/node_modules/expo-camera/ios/Common/ExpoCameraUtils.swift b/node_modules/expo-camera/ios/Common/ExpoCameraUtils.swift
index 68ef7cf..0969fd9 100644
--- a/node_modules/expo-camera/ios/Common/ExpoCameraUtils.swift
+++ b/node_modules/expo-camera/ios/Common/ExpoCameraUtils.swift
@@ -142,20 +142,6 @@ struct ExpoCameraUtils {
     return renderer.image { ctx in
       UIColor.black.setFill()
       ctx.fill(rect)
-      let currentDate = Date()
-      let dateFormatter = DateFormatter()
-      dateFormatter.dateFormat = "dd.MM.YY HH:mm:ss"
-      let text = dateFormatter.string(from: currentDate)
-      text.draw(
-        with: CGRect(
-          x: size.width * 0.1,
-          y: size.height * 0.9,
-          width: size.width,
-          height: size.height
-        ),
-        attributes: [.font: UIFont.systemFont(ofSize: 18), .foregroundColor: UIColor.orange],
-        context: nil
-      )
     }
   }
 
